src/taskpane/pages/login.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  IStackStyles,
+  DefaultPalette,
+  IImageProps,
+  ImageFit,
+  IStackTokens,
+  Stack,
+  FontSizes,
+  PrimaryButton,
+  Link,
+} from "@fluentui/react";
+import { Form, Formik } from "formik";
+import React from "react";
+import { useHistory, withRouter } from "react-router-dom";
+import { useLoginMutation } from "../../generated/graphql";
+import { InputField } from "../components/InputField";
+import Wrapper from "../components/Wrapper";
+
+// Styles definition
+const stackStylesHeader: IStackStyles = {
+  root: {
+    background: DefaultPalette.white,
+  },
+};
+
+const stackStyles: IStackStyles = {
+  root: {
+    margin: 20,
+    marginTop: 30,
+    overflow: "hidden",
+  },
+};
+
+// Logo
+const imageProps: IImageProps = {
+  imageFit: ImageFit.contain,
+  src: "../../assets/jabref.svg",
+};
+
+// Tokens definition
+const verticalGapStackTokens: IStackTokens = {
+  childrenGap: 8,
+  padding: 30,
+};
+
+interface loginProps {}
+
+const Login: React.FC<loginProps> = () => {
+  const history = useHistory();
+  const [loginMutation] = useLoginMutation();
+  return (
+    <Wrapper>
+      <Formik
+        initialValues={{ email: "", password: "" }}
+        onSubmit={async (value, { setErrors }) => {
+          const response = await loginMutation({ variables: value });
+          if (response.data?.login.__typename === "User") {
+            history.push({ pathname: "/" });
+          } else {
+            setErrors({
+              email: "Wrong Email",
```

Comment:
Please show the login error message either above or below the login form (as we have talked about in the last call).